### DISCUSSION TITLE
capture the Touch events

### DISCUSSION THREAD

**Discussion by sanam-bahadur** [yume-chan/ya-webadb#481] [Q&A]
Hi,
Is there any way to listen to the touch events of mobile in scrcpy? For ex- If I click or touch any icon in Mobile side (say bluetooth/anyIcon in slider) then I want to know in App side that this event is being triggered. Thanks.

**Comment by yume-chan** [comment]
No. User inputs are highly sensitive and well protected. Only the foreground App and Accessibility Service can read the inputs.
